.env.example

```
@@ -207,4 +207,7 @@ ML_REFEREE_TRIGGER_CID=
 ML_REFEREE_FINDER_RESULT=
 
 # Temp var for metering comms between nextjs api backend/nodes backend
-INTERNAL_SERVICE_SECRET=psssssssssst
\ No newline at end of file
+INTERNAL_SERVICE_SECRET=psssssssssst
+
+#SCIWEAVE GOOGLE CREDS
+SCIWEAVE_GOOGLE_CLIENT_ID=
```

dotenv-linter (3.3.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)